.travis.yml

```diff
@@ -1,14 +1,37 @@
+---
+os: osx
 sudo: true
-git:
-  depth: 10
+# lie to make travis work with nvm on osx
+language: cpp
+env:
+  matrix:
+    - TRAVIS_NODE_VERSION="0.10"
+    - TRAVIS_NODE_VERSION="0.12"
+    - TRAVIS_NODE_VERSION="2"
+    - TRAVIS_NODE_VERSION="4"
+    - TRAVIS_NODE_VERSION="5"
```

Comment:
Just a side note that I am an OSX user myself and it has been working flawlessly for me.

If you want to test if it's getting `$PATH` correctly, I would do something like this
```
PATH="" apm test
```
then I would spawn `sass-lint` from my package to see if it's working well